package.json

```diff
@@ -37,11 +37,12 @@
     "cors": "^2.8.5",
     "errorhandler": "^1.5.1",
     "express": "^4.17.1",
-    "fhir-works-on-aws-interface": "^9.1.0",
+    "fhir-works-on-aws-interface": "^10.0.0",
     "flat": "^5.0.0",
     "http-errors": "^1.8.0",
     "lodash": "^4.17.15",
     "mime-types": "^2.1.26",
+    "path-parse": "^1.0.7",
```

Comment:
This was a mistake from a my dependency fix, could you please remove it?

Reply:
done